marimo/_cli/convert/commands.py

```diff
@@ -50,6 +50,9 @@ def convert(
     you may need to fix errors like multiple definition errors or cycle
     errors.
     """
+    if not Path(filename).exists():
```

Comment:
you can technically convert from a remote URL, so this would make that fail unfortunately. can we do this check elsewhere? i will also add a test so this is more apparent